keyboards/bastardkb/tbk/config.h

```diff
@@ -0,0 +1,39 @@
+/* 
+ * Copyright 2021 Quentin LEBASTARD <qlebastard@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+#include "config_common.h"
+#define VENDOR_ID 0xA8F8
+#define PRODUCT_ID 0x1828
+#define DEVICE_VER 0x0001
+#define MANUFACTURER Bastard Keyboards
+#define PRODUCT The Bastard Keyboard
+
+#define MATRIX_ROWS 10
+#define MATRIX_COLS 6
+#define DIODE_DIRECTION ROW2COL
+#define MATRIX_ROW_PINS { D7, B5, F7, F6, B6 }
+#define MATRIX_COL_PINS { B4, E6, C6, B1, B3, B2 }
+#define RGB_DI_PIN D2
+#define RGBLED_NUM 38
+#define RGBLIGHT_SPLIT
```

Comment:
Ideally, should set the split distribution. 
```suggestion
#define RGBLED_NUM 38
#define RGBLED_SPLIT { 19, 19 }
#define RGBLIGHT_SPLIT
```

Or is this 38 on each side?  If so: 
```suggestion
#define RGBLED_NUM 76
#define RGBLED_SPLIT { 38, 38 }
#define RGBLIGHT_SPLIT
```